Pattern: About - Image on the right, heading and paragraph

shail-mehta opened the pull request:
Fixes: https://github.com/WordPress/twentytwentyfive/issues/65 

### Screenshot: 
![about-the-event](https://github.com/user-attachments/assets/e1384c7e-fc37-41d6-b336-e04b991a0653)

### Instructions: 
- Create a page.
- Add the pattern.
- Confirm that it's looking like the designs (spacing/colors/text etc).

carolinan commented:
Hi!
Please update the PR with the latest changes from trunk.

Then, remove the background color and text colors: both the attributes in the block comments and the class name.
And then apply the colors by using the new dark and white section style, **style 4.**

In other words, the color declarations should be removed and replaced with the CSS class name that WordPress uses to apply the section style.

`"className":"is-style-section-4"`

---
Tip:
Insert the pattern
Select the outer group block and remove the background color. Apply the section style in the group block options.
Select the columns and remove the text color.

Now you have all the changes you need and you can copy the new class name into the existing pattern file.
Remember that you must not include the meta data.

shail-mehta commented:
Thanks for the feedback. 

I have Rebased current branch & used section style & also removed contrast & base color for text color & background color.

carolinan commented:
Thank you!
